test/faktory_worker/pool_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule FaktoryWorker.PoolTest do
+  use ExUnit.Case
+
+  alias FaktoryWorker.Pool
+
+  describe "child_spec/1" do
+    test "should return a default child_spec" do
+      opts = [name: FaktoryWorker]
```

Comment:
Is a name required when starting the pool?

Reply:
Yes, the pool and push pipeline expect there to always be a name set in opts. This is hidden away when using the library as a client and is handled here https://github.com/SeatedInc/faktory_worker/pull/11/files#diff-ce46dbd5418b6ea656d47b5f554601f3R9